UPGRADE-7.1.md

```diff
@@ -27,14 +59,31 @@ ExpressionLanguage
  * Deprecate passing `null` as the allowed variable names to `ExpressionLanguage::lint()` and `Parser::lint()`,
    pass the `IGNORE_UNKNOWN_VARIABLES` flag instead to ignore unknown variables during linting
 
+Form
+----
+
+ * Deprecate not configuring the `default_protocol` option of the `UrlType`, it will default to `null` in 8.0 (the current default is `'http'`)
+
 FrameworkBundle
 ---------------
 
  * Mark classes `ConfigBuilderCacheWarmer`, `Router`, `SerializerCacheWarmer`, `TranslationsCacheWarmer`, `Translator` and `ValidatorCacheWarmer` as `final`
+ * Deprecate the `router.cache_dir` config option, the Router will always use the `kernel.build_dir` parameter
+
+Intl
+----
+
+ * Extracted `EmojiTransliterator` to a separate component, the new FQCN is `Symfony\Component\Emoji\EmojiTransliterator`
+
+Mailer
+------
+
+ * Postmark's "406 - Inactive recipient" API error code now results in a `PostmarkDeliveryEvent` instead of throwing a `HttpTransportException`
```

Comment:
I'm not sure whether we should classify this as a BC BREAK or not. Technically, if you now handled this in an exception handler your code will be broken. On the other hand, anything that has proper handling now and was an error before is often called a feature.